src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/snippet/de-DE.json

```
@@ -28,6 +28,11 @@
             "tryAgainLater": "Bitte versuche es später erneut.",
             "noConnectionMessage": "Verbindungsprobleme, die gewünschte Sprache kann derzeit nicht installiert werden."
         },
+        "introduction": {
+          "modalTitle": "Willkommen",
+          "headlineWelcome": "Willkommen in der Shopware 6 Administration",
```

```suggestion
          "headlineWelcome": "Willkommen in der Administration von Shopware 6",
```

Otherwise you'd need to do "Shopware-6-Administration", which is ugly af